### PR TITLE
OSDOCS-3138: Removing scheduler policy

### PR DESCRIPTION
**Pull request by bergerhoffer**
https://issues.redhat.com/browse/OSDOCS-3138

Preview: https://deploy-preview-40214--osdocs.netlify.app/openshift-enterprise/latest/nodes/scheduling/nodes-scheduler-about.html

### CODE REVIEW

File: nodes/scheduling/nodes-scheduler-about.adoc [hunk 1]
```diff
@@ -5,42 +5,47 @@ include::modules/common-attributes.adoc[]
...
-// the assembly. Include any combination of concept, procedure, or reference
-// modules required to cover the user story. You can also include other
-// assemblies.
-
 Default pod scheduling::
-{product-title} comes with a xref:../../nodes/scheduling/nodes-scheduler-default.adoc#nodes-scheduler-default[default scheduler] that serves the needs of most users. The default scheduler uses both inherent and customization tools to determine the best fit for a pod.
+{product-title} comes with a default scheduler that serves the needs of most users. The default scheduler uses both inherent and customization tools to determine the best fit for a pod.
+
+////
+// TODO: Determine if we want to pull in any info about the default scheduler from https://docs.openshift.com/container-platform/4.9/nodes/scheduling/nodes-scheduler-default.html
```
Comment (bergerhoffer) [inline nodes/scheduling/nodes-scheduler-about.adoc:19]: @damemi With this PR, I'm removing this page: https://docs.openshift.com/container-platform/4.9/nodes/scheduling/nodes-scheduler-default.html

I wanted to see whether there was anything worthwhile on this page on how the default scheduler works, that should be incorporated elsewhere in the docs? For example, this explanation here:

> The default OpenShift Container Platform pod scheduler is responsible for determining placement of new pods onto nodes within the cluster. It reads data from the pod and tries to find a node that is a good fit based on configured policies. It is completely independent and exists as a standalone/pluggable solution. It does not modify the pod and just creates a binding for the pod that ties the pod to the particular node.

Or this "Understanding default scheduling" section here?

https://docs.openshift.com/container-platform/4.9/nodes/scheduling/nodes-scheduler-default.html#nodes-scheduler-default-about_nodes-scheduler-default

**Comment by bergerhoffer** [comment]
@damemi @kasturinarra Can you please review?

Preview: https://deploy-preview-40214--osdocs.netlify.app/openshift-enterprise/latest/nodes/scheduling/nodes-scheduler-about.html

File: nodes/scheduling/nodes-scheduler-about.adoc [hunk 2]
```diff
@@ -5,42 +5,47 @@ include::modules/common-attributes.adoc[]
...
+The default OpenShift Container Platform pod scheduler is responsible for determining placement of new pods onto nodes within the cluster. It reads data from the pod and tries to find a node that is a good fit based on configured policies. It is completely independent and exists as a standalone/pluggable solution. It does not modify the pod and just creates a binding for the pod that ties the pod to the particular node.
+
+and
+
+https://docs.openshift.com/container-platform/4.9/nodes/scheduling/nodes-scheduler-default.html#nodes-scheduler-default-about_nodes-scheduler-default
+////
 
 Advanced pod scheduling::
-In situations where you might want more control over where new pods are placed, the {product-title} advanced scheduling features allow you to configure a pod so that the pod is required or has a preference to run on a particular node or alongside a specific pod by.
+In situations where you might want more control over where new pods are placed, the {product-title} advanced scheduling features allow you to configure a pod so that the pod is required or has a preference to run on a particular node or alongside a specific pod.
```
Comment (michaelryanpeter) [inline nodes/scheduling/nodes-scheduler-about.adoc:29]: This sentence is longer than ISG's recommended 25 words. Also, I think this goes against the recommended usage of _allow_. Does it make sense to rewrite? 

Perhaps: "You can use the {product-title} advanced scheduling features if you want more control over the placement of new pods. You can configure the affinity of a pod to make it required, define a preference for a pod to run on or alongside a particular node." 

I'm not happy with that rewording of the second sentence. To me, it does not clarify the links in the xrefs below. I wonder if it is necessary?

**Review by michaelryanpeter — COMMENTED** [review]
I made a comment regarding one sentence that is over the ISG's recommended 25 words. I hope the suggestion helps. Please reach out if you have questions.

File: nodes/scheduling/nodes-scheduler-about.adoc (hunk 1 above)
Comment (damemi) [reply nodes/scheduling/nodes-scheduler-about.adoc:19]: @bergerhoffer yeah, I think that pulling in that general info is a good idea. That should still be documented somewhere

**Review by damemi — COMMENTED** [review]
This looks pretty good to me. We should be removing everything that refers to Policy, including:
* the `spec.Policy` CRD field (no longer does anything after https://github.com/openshift/cluster-kube-scheduler-operator/pull/390)
* sample Policy config files
* references to Predicates and Priorities (which were Policy terms)

We may want to list what the default Plugins are in the scheduler now (I see that this removes the pages describing the default predicates/priorities). However, with the current hardcoded Profiles, the goal is to have them represent intent rather than specific sets of plugins. So, the information about default Plugins may not be relevant for now.

If we do end up allowing custom Profile config in the future (for example, with my proposal https://docs.google.com/document/d/15DP15_kNWF354_O0tR2EP0qrvLCjTUJhXx02szwDGXQ/edit?usp=sharing), we might end up wanting to add back some similar pages to the ones that are being removed. But removing them now is the right move, since the current information is outdated anyway.

**Comment by kasturinarra** [comment]
@bergerhoffer i have reviewed the doc PR and added few comments. Have asked jan to check & confirm as well, thanks !!

**Comment by bergerhoffer** [comment]
PR has been updated, details in the Jira, thanks!

File: nodes/scheduling/nodes-scheduler-about.adoc [hunk 3]
```diff
@@ -6,42 +6,37 @@ include::modules/common-attributes.adoc[]
 
 toc::[]
 
-
```
Comment (mburke5678) [inline nodes/scheduling/nodes-scheduler-about.adoc:9]: @bergerhoffer Can/Should you change this heading to match the topic map: _About pod placement using the scheduler_?

File: nodes/scheduling/nodes-scheduler-about.adoc [hunk 4]
```diff
@@ -88,3 +83,11 @@ xref:../../nodes/scheduling/nodes-scheduler-pod-affinity.adoc#nodes-scheduler-po
 
 These advanced scheduling features allow administrators
 to specify which node a pod can be scheduled on and to force or reject scheduling relative to other pods.
+
```
Comment (mburke5678) [inline nodes/scheduling/nodes-scheduler-about.adoc:86]: I like the idea of having this as a separate topic. But I defer. 
Should this come before _Scheduler use cases_? First the default scheduler, then the advanced?

**Comment by mburke5678** [comment]
@bergerhoffer A couple of random thoughts. Feel free to ignore. Otherwise LGTM!

File: nodes/scheduling/nodes-scheduler-about.adoc (hunk 4 above)
Comment (bergerhoffer) [reply nodes/scheduling/nodes-scheduler-about.adoc:86]: Good suggestion, makes sense to me, I'll move it!

File: nodes/scheduling/nodes-scheduler-about.adoc (hunk 3 above)
Comment (bergerhoffer) [reply nodes/scheduling/nodes-scheduler-about.adoc:9]: Good question. I'm not sure why they're different. There are quite a few files that link to this section, so I'll probably leave it alone for now. But let's think about whether one heading makes sense better over the other, and we can do an update later if we want.

**Pull request merged**